l10n_it_fatturapa_in/tests/fatturapa_common.py

```diff
@@ -294,7 +300,24 @@ def run_wizard(
             for wiz_field, wiz_value in wiz_values.items():
                 setattr(wizard_form, wiz_field, wiz_value)
             wizard = wizard_form.save()
-            return wizard.importFatturaPA()
+            res = wizard.importFatturaPA()
+
+            # NEW WIZARD INTERMEDIARY CREATION BASED ON VAT CHECK
+            # Case where the confirmation wizard is shown
+            if res.get("res_model") == "wizard.check.intermediary":
+
+                wizard = self.env["wizard.check.intermediary"].search(
+                    [], order="id desc", limit=1
+                )
```

Comment:
In `res` c'è già tutto quello che serve per trovare il wizard corretto:
```suggestion
                wizard = self.env["wizard.check.intermediary"].browse(res["res_id"])
```

Reply:
:+1: